bin/imsim.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+"""
+This is the imSim program, used to drive GalSim to simulate the LSST.  Written
+for the DESC collaboration and LSST project.  This version of the program can
+read phoSim instance files as is. It leverages the LSST Sims GalSim interface
+code found in sims_GalSimInterface.
+"""
+from __future__ import absolute_import, print_function
+import os
+import sys
+import logging
+import argparse
+
+from lsst.obs.lsstSim import LsstSimMapper
+
+from lsst.sims.photUtils import LSSTdefaults
+from lsst.sims.utils import ObservationMetaData
+
+from lsst.sims.GalSimInterface.galSimCatalogs import GalSimBase
+from lsst.sims.GalSimInterface import GalSimStars
+from lsst.sims.GalSimInterface import GalSimGalaxies
+
+import desc.imsim
```

Comment:
It should fail because the user's PYTHONPATH won't contain the directory with that module, but if you do
```
export PYTHONPATH=<imSim dir>/python/desc/imsim:${PYTHONPATH}
```
then I think it should work.